pennylane/measurements/sample.py

```diff
@@ -210,6 +196,36 @@ def sample(
     Raises:
         ValueError: Cannot set wires if an observable is provided
 
+    .. warning::
+
+        In v0.42, a breaking change removed the squeezing of singleton dimensions.
+        For the qnode:
+
+        >>> @qml.qnode(qml.device('default.qubit'))
+        ... def circuit(wires):
+        ...     return qml.sample(wires=wires)
+
+        We previously squeezed out singleton dimensions like:
+
+        >>> qml.set_shots(circuit, 1)(wires=1)
+        array(0)
+        >>> qml.set_shots(circuit, 2)(0)
+        array([0, 0])
+        >>> qml.set_shots(circuit, 1)((0,1))
+        array([0, 0])
+
+        But now the above circuit will **always** return an array of shape ``(shots, num_wires)``.
+
+        >>> qml.set_shots(circuit, 1)(wires=1)
+        array([[0]])
+        >>> qml.set_shots(circuit, 2)(0)
+        array([[0],
+        [0]])
```

Comment:
```suggestion
        >>> qml.set_shots(circuit, 2)(0)
        array([[0], [0]])
```

Is that actually how it prints? 🤔 ignore suggestion if so